docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -395,15 +395,15 @@ If you run into trouble while installing Great Expectations in Cloud Composer, t
 
 Cloud Composer uses Cloud Storage to store Apache Airflow DAGs (also known as workflows), with each Environment having an associated Cloud Storage bucket (typically the name of the bucket will follow the pattern `[region]-[composer environment name]-[UUID]-bucket`).
 
-The simplest way to perform the migration is to move the entire local `great_expectations/` folder from [Part 1](#part-1-local-configuration-of-great-expectations-that-connects-to-google-cloud-platform) to the Cloud Storage bucket where Composer can access the configuration.
+The simplest way to perform the migration is to move the entire local `gx/` folder from [Part 1](#part-1-local-configuration-of-great-expectations-that-connects-to-google-cloud-platform) to the Cloud Storage bucket where Composer can access the configuration.
 
 First open the Environments page in the Cloud Console, then click on the name of the environment to open the Environment details page. In the Configuration tab, the name of the Cloud Storage bucket can be found to the right of the DAGs folder.
 
-This will take you to the folder where DAGs are stored, which can be accessed from the Airflow worker nodes at: `/home/airflow/gcsfuse/dags`. The location we want to uploads `great_expectations/` is **one level above the `/dags` folder**.
+This will take you to the folder where DAGs are stored, which can be accessed from the Airflow worker nodes at: `/home/airflow/gcsfuse/dags`. The location we want to uploads `gx/` is **one level above the `/dags` folder**.
```

Comment:
I think we have the wrong verb tense for "uploads" here.  At the very least, the second sentence doesn't read right to me.  Are we trying to instead say:
```suggestion
This will take you to the folder where DAGs are stored, which can be accessed from the Airflow worker nodes at: `/home/airflow/gcsfuse/dags`. The location we want to upload the `gx/` folder to is **one level above the `/dags` folder**.
```